build.fsx

```diff
@@ -183,83 +191,204 @@ Target "ReleaseGitHub" (fun _ ->
 
-Target "NuGet.AddNetCore" (fun _ ->
-    if not isDotnetSDKInstalled then failwith "You need to install .NET core to publish NuGet packages"
-    for proj in netcoreSrcFiles do
-        DotNetCli.Pack (fun c -> { c with Project = proj })
+// Build a NuGet package
+// --------------------------------------------------------------------------------------
 
-    let nupkg = sprintf "../../bin/Argu.%s.nupkg" (release.NugetVersion)
-    let netcoreNupkg = sprintf "bin/Release/Argu.%s.nupkg" (release.NugetVersion)
+Target "NuGetPackage" (fun _ ->    
+    if isTravisCI then traceImportant "skipping packaging on TravisCI" else // travis can't package properly, makes mdbs instead of pdbs
+    Paket.Pack (fun p -> 
+    { p with 
+        TemplateFile = __SOURCE_DIRECTORY__ </> "src" </> "Argu" </> "paket.template"
+        WorkingDir = __SOURCE_DIRECTORY__ 
+        OutputPath = __SOURCE_DIRECTORY__ </> pkgDir 
+        ToolPath = __SOURCE_DIRECTORY__ </> ".paket/paket.exe" 
+        Version = release.NugetVersion
+        ReleaseNotes = toLines release.Notes })
+)
 
-    Shell.Exec("dotnet", sprintf """mergenupkg --source "%s" --other "%s" --framework netstandard1.6 """ nupkg netcoreNupkg, "src/Argu/") |> assertExitCodeZero
+Target "MergeDotnetCoreIntoNuget" (fun _ ->
+    if isTravisCI then traceImportant "skipping packaging on TravisCI" else // travis can't package properly, makes mdbs instead of pdbs
+    ensureDirectory pkgDir
+    let nupkg = sprintf "nupkg/Argu.%s.nupkg" release.NugetVersion |> FullName
+    let runTool = runCmdIn "tools" dotnetExePath
+    runTool """mergenupkg --source "%s" --other "%s" --framework netstandard1.6 """ nupkg netcoreNupkg
 )
 
 
+Target "NuGetPush" (fun _ -> Paket.Push (fun p -> { p with WorkingDir = pkgDir }))
+
 Target "Release" DoNothing
 
 // --------------------------------------------------------------------------------------
 // Run all targets by default. Invoke 'build <Target>' to override
 
 Target "Prepare" DoNothing
 Target "PrepareRelease" DoNothing
+Target "NetFramework" DoNothing
+Target "DotNetCore" DoNothing
 Target "Default" DoNothing
 
 "Clean"
   ==> "AssemblyInfo"
-  ==> "SetVersionInProjectJSON"
   ==> "Prepare"
+  =?> ("Build.Net35", not isTravisCI) //mono 4.x doesnt have FSharp.Core 2.3.0.0 installed
   ==> "Build.Net40"
+  ==> "Build.Net461"
   ==> "RunTests"
+  ==> "NetFramework"  
+  
+"Clean"  
+  ==> "InstallDotNetCore"
+  ==> "DotnetRestoreTools"
+  ==> "DotnetRestore"
+  ==> "DotnetBuild"
+  ==> "DotnetRestoreTests"
+  ==> "DotnetBuildTests"
+  ==> "DotNetCore"
+  
+"NetFramework"
+  ==> "NugetPackage" 
+  ==> "DotNetCore"
+  ==> "DotnetPackage" 
+  ==> "MergeDotnetCoreIntoNuget"
   ==> "Default"
 
 "Default"
   ==> "PrepareRelease"
   ==> "SourceLink"
-  =?> ("Build.Net35", not isTravisCI) //mono 4.x doesnt have FSharp.Core 2.3.0.0 installed
-  =?> ("Build.NetCore", isDotnetSDKInstalled)
-  =?> ("RunTests.NetCore", isDotnetSDKInstalled)

```

Comment:
the .net core test `RunTests.NetCore` arent run anymore (checked with local run)